DrawerKit/DrawerKit/Public API/Structs/DrawerConfiguration.swift

```diff
@@ -82,6 +82,9 @@ public struct DrawerConfiguration {
     /// When `false`, the presentation is always to full screen and there is no
     /// partially expanded state. The default value is `true`.
     public var supportsPartialExpansion: Bool
+    
+    /// When `false`, collapsed drawer is never dismissed.
```

Comment:
This needs a comment about the default value (which is `true`). Something like:

/// When `false`, collapsed drawer is never dismissed. The default value is `true`.